modules/300-prometheus/template_tests/copy_custom_certificate_test.go

```diff
@@ -25,18 +25,6 @@ import (
 
 const globalValues = `
 enabledModules: ["vertical-pod-autoscaler-crd", "prometheus"]
-modulesImages:
```

Comment:
don't need here

Reply:
done

Reply:
reverted due to failed test
https://github.com/deckhouse/deckhouse-test-1/actions/runs/3154314187/jobs/5131887278